### ISSUE TITLE
Missing project name Keys file

### ISSUE THREAD

**Issue by RuiAAPeres** [orta/cocoapods-keys#28]
After doing `pod install` with this:
```
plugin 'cocoapods-keys', {
  :project => "Stores Locator",
  :keys => [
    "ParseClientKey",
    "ParseApplicationID",
  ]
}
```

I was expecting:

> will add a new Objective-C class to your Pods xcworkspace. 

But this particular class is missing. I tried to mimic your Eidolon project (cc @ashfurrow ) structure (mine is in Swift as well), but the <Project Name>Keys class does not exist.

**Comment by orta** [comment]
do you need to do a `bundle exec pod install` ?

**Comment by RuiAAPeres** [comment]
@orta I haven't used a gem to specify the cocoapods version.

**Comment by ashfurrow** [comment]
Really? The Keys group is at the bottom. It's not there for you? 

![screen shot 2015-01-27 at 7 13 33 pm](https://cloud.githubusercontent.com/assets/498212/5924050/a6b4fbbe-a658-11e4-81da-6a889f5c09e3.png)

**Comment by orta** [comment]
and you're definitely using CocoaPods 0.36?

**Comment by RuiAAPeres** [comment]
Yes: 

```
0.36.0.beta.1

[!] [Hooks] The use of hooks without specifying a `plugin_name` has been deprecated.
```

**Comment by orta** [comment]
Could be the space in the Project? Seems a bit odd but possible. You might have to run keys from a local folder and add some `puts "1"` values to see where it stops.

If it doesn't say:

```
Generating Pods project
Integrating client project
Adding keys
```

In your pod install then we're not getting anywhere ( or it's having serious keychain issues )

**Comment by RuiAAPeres** [comment]
@orta after a bit more digging, I get at:

```
Analyzing dependencies
Downloading dependencies
Using Alamofire (1.1.3)
Using SwiftyJSON (2.1.3)
Generating Pods project
Integrating client project
Adding keys
```

```
### Error

```
Errno::ENOENT - No such file or directory @ dir_s_mkdir - /Users/ruiperes/Desktop/StoresLocator/Pods/Headers/Public/CocoaPods-Keys
/Users/ruiperes/.rvm/gems/ruby-2.1.1/bundler/gems/cocoapods-keys-81b9301700ed/lib/installer.rb:25:in `mkdir'
/Users/ruiperes/.rvm/gems/ruby-2.1.1/bundler/gems/cocoapods-keys-81b9301700ed/lib/installer.rb:25:in `block in install!'
/Users/ruiperes/.rvm/gems/ruby-2.1.1/bundler/gems/CocoaPods-0740ff3a8310/lib/cocoapods/user_interface.rb:49:in `section'
/Users/ruiperes/.rvm/gems/ruby-2.1.1/bundler/gems/cocoapods-keys-81b9301700ed/lib/installer.rb:15:in `install!'
```

**Comment by RuiAAPeres** [comment]
@orta does it make sense to move this issue to the cocoapods repo?

**Comment by orta** [comment]
No, this is a keys issue, I think the work @ashfurrow is doing today on implementing real frameworks support should fix this. Worth keeping open for prosperity. Maybe this works on eidolon because it used to have lib pods.

**Comment by RuiAAPeres** [comment]
@orta I will try one more time on my home computer and let you guys know if it worked.

**Comment by ashfurrow** [comment]
:+1:

**Comment by RuiAAPeres** [comment]
@orta @ashfurrow tried on my personal laptop, your project (eidolon) works perfectly. Mine doesn't, with the same error. It seems the `Headers/Public` and `Headers/Private` don't exist in my case.

**Comment by orta** [comment]
this could make sense, maybe they only exist at all if you have a single obj-c pod!

**Comment by segiddins** [comment]
@orta I think their existence might be a holdover from a 0.35 installation?

**Comment by orta** [comment]
This should be fixed in master now with #31
